index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <header class="header">
+        <a class="logo" href="">My Logo</a>
+        <nav>
+            <img class="onion" src="image/Union.png" alt="" />
+        </nav>
+    </header>
+
+
+    <div class="container">
+
+        <section class="backgroundimg">
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly work on the design of the website so that it matches the Figma Design 👍🏼 
  - Kindly add some margin around the header.
  - Try to Work on the background image to look the same as in Figma Design.

Your Design:
![image](https://user-images.githubusercontent.com/63473584/185754171-a83bcdc4-a5fc-4539-a4b3-2e851731dfeb.png)

Figma Design:
![image](https://user-images.githubusercontent.com/63473584/185754198-82282968-86f8-4a24-b1b5-4d55671ebe13.png)

You can customize this example as it might help 👇🏼 

```
.backgroundimg {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  padding: 120px 20px;
  background: url('url(/image/background.png)') 0 50px no-repeat;
  background-size: 100% auto;
  background-color: white;
  border-bottom-left-radius: 150px;
  min-height: 718px;
  margin-bottom: 50px;
}
```